test-harness/index.ts

```diff
@@ -76,10 +75,12 @@ describe('cli e2e tests', () => {
         windowsVerbatimArguments: false,
       });
 
+      const expectedStatus = replaceVars(scenario.status.trim(), replacements);
+      const expectedStdout = replaceVars(scenario.stdout.trim(), replacements);
+      const expectedStderr = replaceVars(scenario.stderr.trim(), replacements);
+      const status = commandHandle.status;
```

Comment:
let's maybe differentiate between `expected` and `actual` to be consistent? here, `actualStatus` would tell more than just `status`